### PR TITLE
[Task #40838] Localisation

### PR DESCRIPTION
**Pull request by Rohupt**
## Related Tickets:
- [Task #40838](https://edu-redmine.sun-asterisk.vn/issues/40838)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by Rohupt** [comment]
ready

File: app/Http/Controllers/LocaleController.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class LocaleController extends Controller
+{
+    public function setLocale($locale)
+    {
+        if (! in_array($locale, ['en', 'vi'])) {
```
Comment (vuminhhieu1311) [inline app/Http/Controllers/LocaleController.php]:
```suggestion
        if (!in_array($locale, ['en', 'vi'])) {
```

File: app/Http/Middleware/Language.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Foundation\Application;
+
+class Language
+{
+
+    public function __construct(Application $app, Request $request)
```
Comment (huyenttm-0901) [inline app/Http/Middleware/Language.php]: bỏ các dòng thừa

**Comment by Rohupt** [comment]
ready

File: app/Http/Controllers/LocaleController.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class LocaleController extends Controller
+{
+    public function setLocale($locale)
+    {
+        if (!in_array($locale, ['en', 'vi'])) {
```
Comment (ghost) [inline app/Http/Controllers/LocaleController.php]: bn bỏ mảng ngôn ngữ vào config nhé.

**Comment by Rohupt** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**